### PR TITLE
download openshift-install using oc

### PR DESCRIPTION
**Pull request by vavuthu**
Fixes: #9736

Signed-off-by: vavuthu [vavuthu@redhat.com](mailto:vavuthu@redhat.com)

### CODE REVIEW

**Review by dahorak — DISMISSED** [review]

**Comment by vavuthu** [comment]
verified FIPS here: https://url.corp.redhat.com/023995b

**Review by dahorak — DISMISSED** [review]

**Comment by vavuthu** [comment]
Verified Azure FIPS job : https://url.corp.redhat.com/de16ee7

File: ocs_ci/utility/utils.py [hunk 1]
```diff
@@ -832,17 +836,22 @@ def get_openshift_installer(
         log.debug(f"Installer exists ({installer_binary_path}), skipping download.")
         # TODO: check installer version
     else:
-        version = expose_ocp_version(version)
+        if config.ENV_DATA.get("fips"):
+            # WA issue https://github.com/red-hat-storage/ocs-ci/issues/9755
+            version = "4.16.0-0.nightly-2024-04-26-145258"
```
Comment (petr-balogh) [inline ocs_ci/utility/utils.py:841]: should this be done only on OCP 4.16? What about upgrade scenario where we install N-1?

Comment (vavuthu) [reply ocs_ci/utility/utils.py:841]: yes, its only for 4.16. didn't test the upgrade scenario and I hope upgrade will be fine.

Comment (vavuthu) [reply ocs_ci/utility/utils.py:849]: done

**Review by petr-balogh — APPROVED** [review]

**Review by dahorak — APPROVED** [review]

**Pull request merged**